lib/bolt/transport/sudoable.rb

```diff
@@ -120,7 +120,8 @@ def run_task(target, task, arguments, options = {})
 
               if conn.run_as && stdin
                 # Inject interpreter in to wrapper script and remove from execute options
-                wrapper = make_wrapper_stringio(remote_task_path, stdin, execute_options[:interpreter])
+                wrapper_path = File.join(dir.to_s, 'wrapper.sh')
```

Comment:
I'd be inclined to pull `'wrapper.sh'` into a var, since we use it twice. It seems silly to make the path twice, but I'd rather do that than manually call `copy_file` and `make_executable`.

Reply:
updated